src/Body/BodyRow.tsx

```diff
@@ -169,12 +170,15 @@ function BodyRow<RecordType extends { children?: readonly RecordType[] }>(
           additionalCellProps = column.onCell(record, index);
         }
 
+        const thComponent = getComponent(['body', 'cell'], 'th');
```

Comment:
`cellComponent` is passed by prop. `thCromponent` should also passed by prop instead.

Reply:
> `cellComponent` is passed by prop. `thCromponent` should also passed by prop instead.

@mellis481 Do you refer the review to make some changes here?

Reply:
@yoyo837 He (@SpaNb4) addressed the above comment in commit 5f5ba18 (right below this thread), right?